discovery-provider/src/tasks/tracks.py

```diff
@@ -196,17 +207,23 @@ def parse_track_event(
 
         # if cover_art CID is of a dir, store under _sizes field instead
         if track_record.cover_art:
-            logger.warning(f"tracks.py | Processing track cover art {track_record.cover_art}")
+            # If CID is in IPLD blacklist table, do not continue with indexing
+            if is_blacklisted_ipld(session, track_record.cover_art):
+                logger.info(f"Encountered blacklisted cover art CID {track_metadata_multihash} in indexing new track")
+                return None
+
+            cover_art_multihash = track_record.cover_art
+            track_record.cover_art = None
```

Comment:
by. setting `track_record.cover_art = None`, the `multihash_is_directory(track_record.cover_art)` check always fails and we never process cover_art. Let's remove this `cover_art_multihash` intermediate step and keep things simple

Reply:
@hareeshnagaraj could this be related to https://github.com/AudiusProject/audius-protocol/pull/764/files?

Reply:
i don't think so, even with that change the current code will request a directory check on a `None` type